lib/hcm2/hcm2.js

```diff
@@ -9,14 +9,18 @@ const pDelay = require('delay')
 const BN = require('../bn')
 
 const testNotes = 'true'
-const getID = () => ""
+let ID = 0
+const getID = () => {
+  const id = ID++
+  return id.toString()
+}
```

Comment:
What happens on a machine restart, is it safe to restart the ID even without restarting the hcm2?

Reply:
Yeah, that's not a problem, we could even always send `'0'`. I'm using distinct IDs to associate replies to requests.